src/fairplay.js

```diff
@@ -71,20 +74,21 @@ const addKey = ({video, contentId, initData, cert, options, getLicense, eventBus
         }
         if (err) {
           reject(err);
-          return;
         }
 
         keySession.update(new Uint8Array(license));
       });
     });
 
-    keySession.addEventListener('webkitkeyadded', (event) => {
-      resolve(event);
+    keySession.addEventListener('webkitkeyadded', () => {
+      resolve();
     });
 
     // for testing purposes, adding webkitkeyerror must be the last item in this method
-    keySession.addEventListener('webkitkeyerror', (event) => {
-      reject(event);
+    keySession.addEventListener('webkitkeyerror', () => {
+      const error = keySession.error;
+
+      reject(`KeySession error: code ${error.code}, systemCode ${error.systemCode}`);
```

Comment:
I'm not sure... do you foresee a situation where it would be? The idea here is that you'd get useful messages out of `error` events thrown by video.js and wouldn't necessarily know you'd get something that has a `code` and `systemCode` property. In other words, is `systemCode` something you'd pass to `player.error()`?